modules/centrifuge_classification.nf

```diff
@@ -0,0 +1,103 @@
+// This file contains workflow to classify with centrifuge
+include { unpack_taxonomy } from '../modules/kraken_server'
+
+process unpack_database {
+    label "process_single"
+    storeDir "${params.store_dir}/centrifuge"
```

Comment:
container / conda declarations probably needed here

Reply:
There aren't conda/docker container definitions in the kraken/taxonomy unpack processes so this is consistent. I've now updated to avoid wget and as such it shouldn't need a container def